`crucible-concurrency-golden-tests` build broken after 04d7c0b

After 04d7c0be9274a8ac904590c2a362985fc33edf87, the `crucible-concurrency-golden-tests` build fails with:

```
$ cabal build crucible-concurrency-golden-tests --constraint="aeson<2.0"
Build profile: -w ghc-8.10.4 -O1
In order, the following will be built (use -v for more details):
 - crucible-concurrency-0.1.0.0 (test:crucible-concurrency-golden-tests) (file test/Main.hs changed)
Preprocessing test suite 'crucible-concurrency-golden-tests' for crucible-concurrency-0.1.0.0..
Building test suite 'crucible-concurrency-golden-tests' for crucible-concurrency-0.1.0.0..
[1 of 1] Compiling Main             ( test/Main.hs, /home/rscott/Documents/Hacking/Haskell/crucible/dist-newstyle/build/x86_64-linux/ghc-8.10.4/crucible-concurrency-0.1.0.0/t/crucible-concurrency-golden-tests/build/crucible-concurrency-golden-tests/crucible-concurrency-golden-tests-tmp/Main.o )

test/Main.hs:61:26: error:
    • No constructor has all these fields: ‘inputFiles’, ‘simVerbose’
    • In the expression:
        defaultCruxOptions
          {Crux.inputFiles = [inFile], Crux.simVerbose = 0,
           Crux.globalTimeout = Just 600, Crux.goalTimeout = Just 600,
           Crux.solver = "yices", Crux.checkPathSat = True}
      In the expression:
        (defaultCruxOptions
           {Crux.inputFiles = [inFile], Crux.simVerbose = 0,
            Crux.globalTimeout = Just 600, Crux.goalTimeout = Just 600,
            Crux.solver = "yices", Crux.checkPathSat = True}, 
         defaultCrucesOptions)
      In an equation for ‘options’:
          options
            = (defaultCruxOptions
                 {Crux.inputFiles = [inFile], Crux.simVerbose = 0,
                  Crux.globalTimeout = Just 600, Crux.goalTimeout = Just 600,
                  Crux.solver = "yices", Crux.checkPathSat = True}, 
               defaultCrucesOptions)
   |
61 |        do let options = (defaultCruxOptions { Crux.inputFiles = [inFile],
   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
```

cc @Ptival

--- a/crucible-concurrency/test/Main.hs
+++ b/crucible-concurrency/test/Main.hs
@@ -58,8 +58,11 @@ testSimulator :: FilePath -> FilePath -> IO ()
 testSimulator inFile outFile =
   do contents <- T.readFile inFile
      withFile outFile WriteMode $ \outh ->
-       do let options = (defaultCruxOptions { Crux.inputFiles = [inFile],
-                                              Crux.simVerbose = 0,
+       do let outOpts = (Crux.outputOptions defaultCruxOptions)
+                          { Crux.simVerbose = 0
+                          }
+              options = (defaultCruxOptions { Crux.outputOptions = outOpts,
+                                              Crux.inputFiles = [inFile],
                                               Crux.globalTimeout = Just 600,
                                               Crux.goalTimeout = Just 600,
                                               Crux.solver = "yices",
@@ -67,5 +70,5 @@ testSimulator inFile outFile =
                                             },
                          defaultCrucesOptions)
           let ?outputConfig = Crux.mkOutputConfig (outh, False) (outh, False)
-                                Crux.cruxLogMessageToSayWhat $ Just (fst options)
+                                Crux.cruxLogMessageToSayWhat $ Just (Crux.outputOptions (fst options))
           run options

